### PR TITLE
Setup create profile button and form functionality

### PR DESCRIPTION
**Pull request by LoganTown**
Hi, messed up the branch and push request etc but I think this should work. Includes code for making a new account and the page/functionality

### CODE REVIEW

**Comment by Curtis-Barnhart** [comment]
What issues is this PR resolving? We need to know what this branch is actually supposed to accomplish before we can review whether or not it's accomplished the goals it is supposed to fulfill.

**Comment by Curtis-Barnhart** [comment]
Also it might be good to note that, per the rules on this page, at least two people need to review this so it might be helpful to assign more than just me under Reviewers.

**Comment by LoganTown** [comment]
I added everyone, no idea how to link specific issues

**Comment by Curtis-Barnhart** [comment]
[Try this](https://www.google.com/search?q=how+to+link+issues+in+github&sca_esv=577979125&source=hp&ei=zEBAZarLL4nHkPIPx8C0uAc&iflsig=AO6bgOgAAAAAZUBO3NhmdtJTujIfJS-B7Kj4DJJ0RMsm&ved=0ahUKEwiq4ZLm-p6CAxWJI0QIHUcgDXcQ4dUDCAs&uact=5&oq=how+to+link+issues+in+github&gs_lp=Egdnd3Mtd2l6Ihxob3cgdG8gbGluayBpc3N1ZXMgaW4gZ2l0aHViMgUQABiABDIGEAAYFhgeMgYQABgWGB4yBhAAGBYYHjIIEAAYigUYhgMyCBAAGIoFGIYDSJEdUPsBWNoccAF4AJABAJgB6QGgAekaqgEHMTAuMTMuNLgBA8gBAPgBAagCCsICEBAAGAMYjwEY5QIY6gIYjAPCAhAQLhgDGI8BGOUCGOoCGIwDwgIREC4YgAQYsQMYgwEYxwEY0QPCAgsQABiKBRixAxiDAcICCxAAGIAEGLEDGIMBwgILEC4YgAQYxwEYrwHCAgsQLhiKBRixAxiDAcICCxAuGIAEGLEDGIMBwgILEC4YgAQYxwEY0QPCAgUQLhiABMICCBAAGIoFGLEDwgIIEAAYgAQYsQPCAggQLhiABBjUAsICBBAAGAM&sclient=gws-wiz).

**Comment by mikeryu** [comment]
STOP WORKING ON 130 IN 105 :(((((

https://tenor.com/bn03J.gif

File: our_site/our_site/settings.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 """
-Django settings for our_site project.
+Django settings for our_site project. haha
```
Comment (Curtis-Barnhart) [inline our_site/our_site/settings.py:2]: As amusing as this is, I am hesitant to say it should stay in the pull request.

**Review by Curtis-Barnhart — CHANGES_REQUESTED** [review]
Creating a new user with a username of greater than 150 characters is allowed, contradicting the message provided on the page that limits your username to 150 characters. Either that text needs to be changed or the text actually has to be limited.
I am not sure what happens when you create a user in the this method - whether the name is chopped off or whether it is just allowed, but either way could be confusing for users.

**Comment by mnugent04** [comment]
> I added everyone, no idea how to link specific issues

all you have to do is in the comments add "#10" or what ever the issue number is

File: our_site/planner/templates/registration/login.html [hunk 2]
```diff
@@ -30,8 +30,13 @@
 
 <input type="submit" value="login">
 <input type="hidden" name="next" value="{{ next }}">
+
+<br> <br>
```
Comment (mnugent04) [inline our_site/planner/templates/registration/login.html:34]: don't think this line is needed not doing anything/ printing anything

**Review by mnugent04 — CHANGES_REQUESTED** [review]

**Comment by Curtis-Barnhart** [comment]
Creating a new user with a username of greater than 150 characters is allowed, contradicting the message provided on the page that limits your username to 150 characters. Either that text needs to be changed or the text actually has to be limited. <-- still needs to be addressed.

**Review by Curtis-Barnhart — APPROVED** [review]

**Review by mnugent04 — APPROVED** [review]

**Pull request merged**